detox/android/detox/src/full/java/com/wix/detox/reactnative/idlingresources/animations/AnimatedModuleIdlingResource.kt

```diff
@@ -0,0 +1,64 @@
+package com.wix.detox.reactnative.idlingresources.animations
+
+import android.util.Log
+import android.view.Choreographer
+import androidx.test.espresso.IdlingResource.ResourceCallback
+import com.facebook.react.animated.NativeAnimatedModule
+import com.facebook.react.bridge.ReactContext
+import com.wix.detox.reactnative.idlingresources.DetoxIdlingResource
+
+/**
+ * Created by simonracz on 25/08/2017.
+ */
+/**
+ *
+ * Espresso IdlingResource for React Native's Animated Module.
+ *
+ *
+ * Hooks up to React Native internals to monitor the state of the animations.
+ *
+ */
+class AnimatedModuleIdlingResource(private val reactContext: ReactContext) : DetoxIdlingResource(),
+    Choreographer.FrameCallback {
+
+    override fun getName(): String {
+        return AnimatedModuleIdlingResource::class.java.name
+    }
```

Comment:
```suggestion
    override fun getName() = AnimatedModuleIdlingResource::class.java.name
```

etc. etc.
I think you should do this all across the board, it makes things much more concise and far less java-like